cmd/vodloadtester/vodloadtester.go

```diff
@@ -0,0 +1,423 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"math/rand"
+	"os"
+	"runtime"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/golang/glog"
+	api "github.com/livepeer/go-api-client"
+	"github.com/livepeer/stream-tester/internal/utils"
+	"github.com/peterbourgon/ff/v2"
+)
+
+type cliArguments struct {
+	Verbosity    int
+	Simultaneous uint
+	Version      bool
+
+	DirectUpload    bool
+	ResumableUpload bool
+	Import          bool
+
+	TaskCheck   bool
+	APIServer   string
+	APIToken    string
+	Filename    string
+	VideoAmount uint
+	OutputPath  string
+
+	TestDuration       time.Duration
+	StartDelayDuration time.Duration
+}
+
+type vodLoadTester struct {
```

Comment:
Could we move this struct to its own package under `internal/app/vodloadtester`? That's the pattern we've been following in this project, and I think it's indeed cleaner to separate the business logic of the load tester from the code that only deals with the "command line interface" parsing/serializing of results etc.